scene/gui/item_list.cpp

```diff
@@ -28,6 +28,8 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 #include "item_list.h"
+#include "joypad.h"
```

Comment:
Though I don't see why you need it? `"os/input.h"` should be enough.

Reply:
@akien-mga for backspace key constant.

Reply:
That's defined in https://github.com/godotengine/godot/blob/30dadb12281924ccb59efc8df4b3b9ee68c6e3a4/core/os/keyboard.h not in joypad.h..

Reply:
Well, Visual Studio suggested joypad.h :) Will fix it now.